.github/workflows/scripts/ti_build/vulkan.py

```diff
@@ -57,4 +61,4 @@ def setup_vulkan():
         os.environ["VK_LAYER_PATH"] = str(prefix / "Bin")
         path_prepend("PATH", prefix / "Bin")
     else:
-        return
+        return
```

Comment:
Have you run `pre-commit` by the way? Run something like 
```
pre-commit run -a
```
I feel like it might have an opinion about linefeeds at the end of python files.